test/integration/targets/prepare_http_tests/tasks/main.yml

```diff
@@ -1,7 +1,7 @@
 # The docker --link functionality gives us an ENV var we can key off of to see if we have access to
 # the httptester container
 - set_fact:
-    has_httptester: "{{ lookup('env', 'HTTPTESTER') != '' }}"
+    has_httptester: lookup('env', 'HTTPTESTER') != ''
```

Comment:
Thanks ! Now that you have spelled it out, it is strange I did not see this. Testing again.